psl/psl/tests/validation/datasource/url_same_as_shadow.prisma

```diff
@@ -0,0 +1,12 @@
+datasource testds {
+    provider = "mysql"
+    url = "mysql://testurl"
+    shadowDatabaseUrl = "mysql://testurl"
+}
+
+// [1;91merror[0m: [1mshadowDatabaseUrl is the same as url for datasource "testds". Please specify a different database as shadow database.[0m
```

Comment:
```suggestion
// [1;91merror[0m: [1mshadowDatabaseUrl is the same as url for datasource "testds". Please specify a different database as shadow database to avoid data loss.[0m
```